documentation-templates/README.md

```diff
@@ -0,0 +1,19 @@
+# documentation-templates
+
+The templates available in [scaffolder-templates](https://github.com/parodos-dev/workflow-software-templates/tree/main/scaffolder-templates) are integrated in the Backstage Orchestrator plugin under `locations` in the following deployment manifest [values](https://github.com/anludke/orchestrator-helm-chart/blob/main/charts/orchestrator/values.yaml#L121).
```

Comment:
pls update the link to the values to point to its new location
https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/rhdh-operator.yaml#L186

Reply:
Fixed.